### ISSUE TITLE
pob Portuguese Brazil subtitles don't show label [Issue]: 

### ISSUE THREAD

**Issue by webysther** [jellyfin/jellyfin#9072]
### Please describe your bug

When adding a Brazilian Portuguese subtitle it fails to translate to the correct label.

![image](https://user-images.githubusercontent.com/750007/211836860-1dee8dae-5059-4a3b-8ace-1f6b68276567.png)

only *por* works:
![image](https://user-images.githubusercontent.com/750007/211837015-81789a23-29b0-474e-ab89-306d819d67be.png)

### Jellyfin Version

10.8.0

### if other:

10.8.7

### Environment

```markdown
- OS: DSM
- Virtualization: Docker
- Clients: All
- Browser: All
- FFmpeg Version: N/A
- Playback Method: N/A
- Hardware Acceleration: N/A
- Plugins: N/A
- Reverse Proxy: N/A
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by webysther** [comment]
not stale

**Comment by crobibero** [comment]
Duplicate of https://github.com/jellyfin/jellyfin/issues/6302

**Comment by curbengh** [comment]
`pob` does not refer to Brazilian Portuguese officially. Existing programs that support only 2 or 3-letter language code adopt it as a workaround. IETF BCP 47 standard uses [`pt-BR`](https://schneegans.de/lv/?tags=pt-BR&Format=text) instead.

**Comment by nandie** [comment]
It still an issue

**Comment by lcdss** [comment]
Open Subtitles use `pob` to refer to `Brazillian Portuguese`, so why not just display a better label, in the case there are no other languages that use the same abbreviation?

**Comment by webysther** [comment]
> Open Subtitles use `pob` to refer to `Brazillian Portuguese`, so why not just display a better label, in the case there are no other languages that use the same abbreviation?

the problem is happen with mostly all languages that uses suffix, no matter if uses pob or pt-BR.

**Comment by webysther** [comment]
The temporary solution I found:

```nginx
location /Users {
  ...
  sub_filter_types application/json;
  sub_filter "Pob - S" "Portuguese - S";
}
```

**Comment by felix920506** [comment]
it was closed as a duplicate of #6302

Please leave any comment you might have over there instead of attempting to revive this issue.

**Comment by webysther** [comment]
> This issue was closed as a duplicate of [#6302](https://github.com/jellyfin/jellyfin/issues/6302)
> 
> Please leave any comment you might have over there instead of attempting to revive this issue.

At least read the another issue before make myself wast my time reading this, thanks.
